common/thrifthelpers/thrift_serializer.go

```diff
@@ -0,0 +1,40 @@
+package thrifthelpers
+
+import "github.com/apache/thrift/lib/go/thrift"
+
+type Serializer interface {
```

Comment:
cool looks like interface isn't necessary then? So either remove the interface or add the structs up to you!